orchestrator/test/test_application_class.py

```diff
@@ -88,9 +84,8 @@ def pidof(app):
             except:
                 return False
 
-    def test_start_stop(self, device: Device, support_app: str):  # noqa
-        uninstall_apk(support_app, device)
-        app = Application.from_apk(support_app, device)
+    def test_start_stop(self, device: Device, app_setup):  # noqa
+        app, _, _ = app_setup
```

Comment:
Can't comment on unchanged lines, but shouldn't the try / finally uninstall bit be removed here too?

Reply:
Also in test_monkey

Reply:
You're right on the test_start_stop. Removing. For test_monkey, I was hesitant because it was using asyncio. I'll try changing it and see it still works.